app-backend/batch/src/main/scala/com/azavea/rf/batch/aoi/UpdateAOIProject.scala

```diff
@@ -237,19 +237,10 @@ final case class UpdateAOIProject(projectId: UUID)(
         affectedRows <- ProjectDao.updateProject(newProject, proj.id, user)
       } yield affectedRows
 
-    def kickoffIngestsIO: ConnectionIO[(UUID, List[UUID])] =
-      for {
-        projId <- addScenesToProjectWithProjectIO
-        sceneIds <- SceneWithRelatedDao.getScenesToIngest(projId) map {
-          (scenes: List[Scene.WithRelated]) =>
-            scenes map { _.id }
-        }
-      } yield (projId, sceneIds)
-
-    val (projId, sceneIds) = kickoffIngestsIO.transact(xa).unsafeRunSync
-    sceneIds map { kickoffSceneIngest }
+    val (projId, nNewScenes) =
```

Comment:
is `nNewScenes` a typo?

Reply:
It is not -- it's like `nAffectedRows` elsewhere. I could make it more obviously not a typical though

Reply:
I don't actually see `nAffectedRows` anywhere, and there's one or two usages of `nAffected`. Stuff like `actionCount` seems to be the more common naming scheme we've used throughout the app.

Reply:
Huh... maybe I wrote `nAffected` once forever ago and it's been in my brain since. You're right, it's nowhere. :ghost: